### PR TITLE
Unit Test PRs.

### PR DESCRIPTION
**Pull request by kfox1111**

### CODE REVIEW

File: .github/workflows/helm-chart-ci.yaml
Comment (edwbuck) [inline .github/workflows/helm-chart-ci.yaml:1]: Should we capture the output into a report that integrates with the badges on our README.md?
We can defer such an action to a later MR, especially if the integration would slow down the merge.

**Review by edwbuck — COMMENTED** [review]
If this is complete, take it out of draft and I've already reviewed it at commit #9.  It will be approved and merged immediately.
If you need to change it in any way, keep it in draft and I'll re-review it with future commits.
If this is going to take time, I'm also open to merging the Makefile changes independently.

File: .github/workflows/helm-chart-ci.yaml
Comment (kfox1111) [reply .github/workflows/helm-chart-ci.yaml:1]: Separate feature? Good to do in a followon?

Comment (edwbuck) [reply .github/workflows/helm-chart-ci.yaml:1]: I'm OK with a follow on item.

**Review by edwbuck — APPROVED** [review]
Approved

**Pull request merged**